#! /bin/sh /usr/share/dpatch/dpatch-run
## 81-patch-idlstubs-order-cossevent.dpatch by Raphael Bossek <bossekr@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch order of idl_stub generation

@DPATCH@

--- ACE_wrappers/TAO/orbsvcs/orbsvcs/GNUmakefile.CosEvent.orig	2004-10-27 19:03:01.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/GNUmakefile.CosEvent	2004-10-27 19:03:19.000000000 +0200
@@ -29,6 +29,10 @@
 endif
 
 CosEvent = \
+  CosEventCommC \
+  CosEventChannelAdminC \
+  CosEventCommS \
+  CosEventChannelAdminS \
   CosTypedEventCommC \
   CosTypedEventChannelAdminC \
   CosTypedEventCommS \
@@ -39,10 +43,6 @@
   CosEvent/CEC_TypedConsumerAdmin \
   CosEvent/CEC_TypedSupplierAdmin \
   CosEvent/CEC_DynamicImplementation \
-  CosEventCommC \
-  CosEventChannelAdminC \
-  CosEventCommS \
-  CosEventChannelAdminS \
   CosEvent/CEC_ConsumerAdmin \
   CosEvent/CEC_ConsumerControl \
   CosEvent/CEC_Default_Factory \
